utils/wasi-nn/install-openvino.sh

```diff
@@ -3,10 +3,10 @@
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
 set -e
-echo "Installing OpenVINO with version 2023.0.2"
+echo "Installing OpenVINO with version 2023.2.0"
 wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
 apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
 echo "deb https://apt.repos.intel.com/openvino/2023 ubuntu20 main" | tee /etc/apt/sources.list.d/intel-openvino-2023.list
 apt update
-apt-get -y install openvino-2023.0.2
+apt-get -y install openvino-2023.2.0
```

Comment:
This is due to:
- https://github.com/openvinotoolkit/openvino/issues/19784

Reply:
(And this can't compiled due the API changes: https://docs.openvino.ai/2022.3/openvino_2_0_transition_guide.html)

Reply:
Hi @AkihiroSuda 
I just created an issue for this. We would like to move forward to a new OpenVINO version. If this blocks you, please temporarily remove the OpenVINO target to make the remaining CI workflows run.

Reply:
Thanks, temporarily removed OpenVINO